utils/oscap-info.c

```diff
@@ -49,6 +49,27 @@
 
 #include "oscap-tool.h"
 
+// COPIED from src/XCCDF/benchmark.c
+struct xccdf_profile * xccdf_benchmark_get_profile_by_id(struct xccdf_benchmark *benchmark, const char *profile_id)
```

Comment:
Please don't copy code around. That is considered a bad practice. 

Perhaps you had decide to copy because had troubles with including and compiling. That's because `xccdf_benchmark_get_profile_by_id`  isn't part of public API and therefore it's hidden for `utils`. You can make the original function public, by moving its declaration into the public API. I think you can move the declaration of this function from `src/XCCDF/item.h` to `src/XCCDF/public/xccdf_benchmark.h`. and  include public/xccdf_benchmark.h here (if its not already included here).